src/extension.ts

```diff
@@ -20,9 +20,10 @@ async function changeTheme(
   const userSettings = getUserSettings();
   const i = getRandomInt(themeList.length - 1);
   const newTheme = themeList[i];
+  const oneMoreTimeThemeList: string[] = <string[]>userSettings.get(SettingsKeys.OneMoreTimeThemeList, MATERIAL_LIST);
```

Comment:
I would change this name in something like `preventReloadThemeList` or something along those lines